code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/IuferevKirillTest.java

```diff
@@ -0,0 +1,34 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class IuferevKirillTest {
+
+    private final double delta = 0.001;
+
+    @Test
+    public void areNumberIsNotEqualComplexNumber() {
+        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
+        assertFalse(complexNumber.equals(24));
+    }
+
+    @Test
+    public void canFormatToString() {
+        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
+        assertEquals("2.4 + 4.4i", complexNumber.toString());
+    }
+
```

Comment:
Expected value should not be a magical constant, create a dedicated variable.
it should look something like this: 
        ComplexNumber complexNumber = new ComplexNumber(2.4, 4.4);
        var expectedString = "2.4 + 4.4i";
        assertEquals(expectedString, complexNumber.toString());

Reply:
fixed